cypress.config.js

```diff
@@ -8,9 +8,14 @@ module.exports = defineConfig({
         generateUser() {
           const randomNumber = Math.ceil(Math.random(1000) * 1000);
           return {
-            username: faker.name.firstName() + `${randomNumber}`,
+            username: faker.name.firstName(),
             email: 'test' + `${randomNumber}` + '@mail.com',
```

Comment:
Consider using a template literal for creating the email string. It would make the code cleaner and easier to read. For example: `email: `test${randomNumber}@mail.com`